.github/workflows/build.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Test with pytest
         if: matrix.python-version != '3.10'
         run: |
-          pytest test
+          pytest test --ignore=test/agentchat/contrib
```

Comment:
Wouldn't this cause all contrib agent test to be ignored?

Reply:
Yes, that is intentional based on previous discussions. https://github.com/microsoft/autogen/pull/857#issuecomment-1837821794

The idea is to disable testing the `contrib` folder, and since tests in that folder require different optional dependencies, `contrib` tests are picked up separately in `.github/workflows/contrib-tests.yml` & `.github/workflows/contrib-openai.yml`.